tests/conftest.py

```diff
@@ -619,6 +619,160 @@ def mentioned_messages_combination(request: Any) -> Tuple[Set[int], Set[int]]:
     return deepcopy(request.param)
 
 
+@pytest.fixture
+def custom_profile_fields_fixture() -> List[Dict[str, Union[str, int]]]:
+    return [
+        {
+            "id": 1,
+            "name": "Phone number",
+            "type": 1,
+            "hint": "",
+            "field_data": "",
+            "order": 1,
+        },
+        {
+            "id": 2,
+            "name": "Biography",
+            "type": 2,
+            "hint": "What are you known for?",
+            "field_data": "",
+            "order": 2,
+        },
+        {
+            "id": 3,
+            "name": "Favorite food",
+            "type": 1,
+            "hint": "Or drink, if you'd prefer",
+            "field_data": "",
+            "order": 3,
+        },
+        {
+            "id": 4,
+            "name": "Favorite editor",
+            "type": 3,
+            "hint": "",
+            "field_data": '{"0":{"text":"Vim","order":"1"},\
+            "1":{"text":"Emacs","order":"2"}}',
```

Comment:
What is the source of these? Is the `\` character for a line break, or in the raw data?

Reply:
It's for a line break, to keep the linter from complaining.